src/main/webapp/createfile.html

```diff
@@ -4,21 +4,27 @@
     <meta charset="UTF-8">
     <title>Create File Configuration</title>
     <link rel="stylesheet" href="assets/createfile.css">
+    <link rel="stylesheet" href="assets/general.css">
   </head>
   <body>
+    <div class="drop_menu">
+			<button class="menu_button">Page Menu</button>
+			<div class="dropdown_content">
+				<a href="readfile.html">Read Configuration</a>
+        <a href="createfile.html">Create Configuration</a>        
+        <a href="editfile.html">Edit Configuration</a>
+        <a href="deletefile.html">Delete Configuration</a>
+		  </div>
+    </div>
     <h1>Billing Configuration Sync Tool</h1><br>
     <h2>Create Configuration</h2>
-    <div id="links">
-      <a href="readfile.html">Read Configuration</a>
-      <a href="editfile.html">Edit Configuration</a>
-      <a href="deletefile.html">Delete Configuration</a>
-    </div><br>
+    <br>
     <div id="createform">
       <form>
-        <span>Customer ID:</span>
+        <label>Customer ID:</label>
```

Comment:
I think the label tags on this pr should be `<label for="customer-id">`

Reply:
I'll add label tags for each <label>